server/src/main/java/org/solarbank/server/service/CalculateService.java

```diff
@@ -0,0 +1,35 @@
+package org.solarbank.server.service;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.solarbank.server.dto.UserInputDto;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculateService {
+    /**
+     * temporary method to return static mock data.
+     *
+     * @param userInputDto data from user input
+     * @return response returns calculated data
+     */
+    public Map<String, Object> processUserInput(UserInputDto userInputDto) {
+
+        Map<String, Double> energyGenPerMonth = new HashMap<>();
+        energyGenPerMonth.put("January", 0.1);
+        energyGenPerMonth.put("February", 0.2);
+
+        Map<String, Object> savingsPerYear = new HashMap<>();
```

Comment:
We should represent currency and amounts using the money api that comes with Java:

https://www.baeldung.com/java-money-and-currency

As you get more experienced you will learn there are two domains which are a nightmare to work with: time and money. It's always best to use the built in apis for these things.

Reply:
now using currency and money api for amount and currnecyCode